runtime/service.go

```diff
@@ -503,10 +502,14 @@ func (s *service) createVM(requestCtx context.Context, request *proto.CreateVMRe
 		return errors.Wrapf(err, "failed to build VM configuration")
 	}
 
-	opts := []firecracker.Opt{
-		firecracker.WithLogger(s.logger),
-	}
+	opts := []firecracker.Opt{}
 
+	if v, ok := s.config.DebugHelper.GetFirecrackerSDKLogLevel(); ok {
+		logger := log.G(s.shimCtx)
+		logrus.SetLevel(v)
```

Comment:
Since we allow SDK and firecracker-containerd to have different log levels, we shouldn't call logrus.SetLevel() to modify the global logger. The fact SDK's log level impacts the global logger and overrides firecracker-containerd's log level regarding the global logger is confusing.

Reply:
Ah yea, that was copy pasta